Gemfile.lock

```diff
@@ -120,22 +122,23 @@ GEM
       sentry-rails (~> 5.3)
       sentry-ruby (~> 5.3)
       statsd-ruby (~> 1.5)
-    govuk_design_system_formbuilder (4.1.1)
+    govuk_design_system_formbuilder (5.0.0)
```

Comment:
Uhh I've noticed this huge bump 😬 I know for a fact frontend 5.0.0 introduces several breaking changes, although not sure if it has an effect in the form builder.

Reply:
It's pulled in as a dependency, presumably through the metadata presenter?

Reply:
Yep, I think to err on the safe side we will need to add a < 5.0.0 in the gem spec.
In the interim we can revert this change in this Gemfile.lock to version 4.1.1 just in case.